### ISSUE TITLE
Zerotier-cli shows offline

### ISSUE THREAD

**Issue by Tennyleaz** [zerotier/ZeroTierOne#1025]
**Describe the bug**
Zerotier-cli shows offline, zerotier GUI shows connected. I cannot ping other zerotier clients. It was fine last month.
I tried to install zerotier one on another PC in same network, and it shows online, but they cannot ping each other via zerotier's IP either.

**Expected behavior**
I expected zerotier-cli shows online, and I should ping other clients.

**Screenshots**
![圖片](https://user-images.githubusercontent.com/7089271/63929819-b0126480-ca84-11e9-8593-cf0e641a174e.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 x64 1903
 - ZeroTier Version 1.4.4
 - Hardware Gigabyte B450i motherboard
 - I am in an office network

**Comment by lunfangyu** [comment]
OS: Windows 10 x64 1909
ZeroTier Version 1.4.6
I have a same problem

**Comment by vakker** [comment]
Same problem.
This usually happens when I'm on network that blocks UDP traffic. I have `"allowTcpFallbackRelay": true` but it shows `"tcpFallbackActive": false`.
Any suggestions on how to debug this issue?

**Comment by graphixillusion** [comment]
I'm getting the same problem under Windows 2004. Any clue?

**Comment by didyouexpectthat** [comment]
I ran into the same thing on Debian 10. I could not get it to work until after I ended up completely removing zerotier and reinstalling, at which point it came back up as online. 

```
root@zerotierbridge:~# zerotier-cli listnetworks
200 listnetworks <nwid> <name> <mac> <status> <type> <dev> <ZT assigned ips>
200 listnetworks $nwid $name $mac OK PRIVATE $dev $ip
root@zerotierbridge:~# zerotier-cli -j status
{
 "address": "$address",
 "clock": 1600133424315,
 "config": {
  "physical": null,
  "settings": {
   "allowTcpFallbackRelay": true,
   "portMappingEnabled": true,
   "primaryPort": 9993,
   "softwareUpdate": "disable",
   "softwareUpdateChannel": "release"
  }
 },
 "online": false,
 "planetWorldId": 149604618,
 "planetWorldTimestamp": 1591823606334,
 "publicIdentity": "$identity",
 "tcpFallbackActive": false,
 "version": "1.4.6",
 "versionBuild": 0,
 "versionMajor": 1,
 "versionMinor": 4,
 "versionRev": 6
}
```

**Comment by mkpb** [comment]
Same here.
Occasionally I have to remove zerotier. Wipe out /var/lib/zerotier-one and reinstall it.

**Comment by mkpb** [comment]
Was thinking about the paid plan ... but I'm worried that I'll get the same instructions from support and nothing more.

**Comment by laduke** [comment]
Are you using version 1.6.2 @mkpb ?

**Comment by graphixillusion** [comment]
@laduke just happened again with version 1.6.1 so nothing changed. I didn't try version 1.6.2 yet becouse the msi download on the website still get the old 1.6.1 version.

**Comment by mkpb** [comment]
> Are you using version 1.6.2 @mkpb ?

I was using 1.6.1 and had problems with some nodes in the mesh weren't seeing each other.
I did apt-get upgrade - it picked up 1.6.2 version and upgraded. It didn't help. Reboot didn't help.

Full reinstall plus removing /var/lib/zerotier-one helped (including authenticating new client in your controller and removing rouge one).

**Comment by mkpb** [comment]
Unfortunately each day of the last two or three weeks our IT team was spending 25% of time reinstating ZT connectivity by a fresh install. What's funny that each time such a fresh install caused the node to work perfectly fine for several hours or even a whole day.

I assume no show stopper was caused by us because a full reinstall ALWAYS repaired connectivity for at least several hours.

I love ZT, but unfortunately, because of numerous problems we recently encountered we just completed a transition of our accounting team to our private "Wireguard via VPS" star topology network... which seems stable for now. 

If WG works as expected more teams will follow.

Wish you all the best with ZT. I'm your fan, but we just lost too much time on maintenance recently.
